BlogTemplate.Tests/Pages/EditTests.cs

```diff
@@ -22,25 +23,24 @@ public void UpdatePost_TitleIsUpdated_UpdateSlug()
 
             Post post = new Post()
             {
+                Title = "Title",
                 Slug = "Title",
+                Body = "Body",
+                Excerpt = "Excerpt",
                 IsPublic = true,
                 PubDate = DateTimeOffset.Now,
             };
 
             testDataStore.SavePost(post);
 
-            EditedPostModel editedPost = new EditedPostModel()
-            {
-                Title = "Edited Title",
-            };
-
-            EditModel model = new EditModel(testDataStore, slugGenerator, excerptGenerator);
+            EditModel testEditModel = new EditModel(testDataStore, slugGenerator, excerptGenerator);
+            testEditModel.PageContext = new PageContext();
+            testEditModel.OnGet(post.Id);

```

Comment:
It's a little weird to call `OnGet()` as part of your setup.  You should be able to just call OnPostPublish in isolation of any other behaviors.